Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputManager.cpp

```diff
@@ -255,6 +255,13 @@ namespace AzToolsFramework
             }
         }
 
+        // If the application state changes (e.g. we have alt-tabbed or minimized the
+        // main editor window) then ensure all input channels are cleared
+        if (eventType == QEvent::ApplicationStateChange)
```

Comment:
do you want to filter to specific application states:

https://doc.qt.io/qt-5/qt.html#ApplicationState-enum

Reply:
I was doing something similar before (please see earlier comment from Nick and my reply) but decided clearing input for any kind of application state change makes sense (at least for now). On desktop most (all?) seem to be the 0x02 option.